test/common.coffee

```diff
@@ -23,6 +23,17 @@ describe 'Common functions', () ->
       assert.equal defaultCam.path, '/onvif/device_service'
       done()
 
+  describe 'autoconnect disabled', () ->
+    it 'should not connect automatically', (done) ->
+      options = {
+        autoconnect: false
+        timeout: 0
+      }
+      testcam = new onvif.Cam options, assert.fail
+      setTimeout () ->
+        done()
```

Comment:
Can you check here that it didn't actually try and connect?

Reply:
in this case, assert.fail() will be called. I've improved the readability now.